taipy/core/_repository/db/_sql_connection.py

```diff
@@ -87,4 +77,4 @@ def _build_connection() -> Connection:
 
 @lru_cache
 def __build_connection(db_location: str):
-    return sqlite3.connect(db_location, check_same_thread=False)
+    return sqlite3.connect(db_location, check_same_thread=False, timeout=200)
```

Comment:
I believe 200 seconds is unreasonable.

Reply:
I agree, I'd say 15 or lower is good enough